tracer/src/Datadog.Tracer.Native/method_rewriter.cpp

```diff
@@ -701,6 +702,76 @@ HRESULT TracerMethodRewriter::Rewrite(RejitHandlerModule* moduleHandler, RejitHa
     return S_OK;
 }
 
+std::tuple<WSTRING, WSTRING>
+TracerMethodRewriter::GetResourceNameAndOperationName(const ComPtr<IMetaDataImport2>& metadataImport,
```

Comment:
@zacharycmontoya I've decided to keep this function around for a future use. In the future we could introduce new integration for trace annotation (TraceAnnotationIntegrationV2) that will load the resource name & operation name in the bytecode instrumentation using `LoadStr`, and ditch the slow path of the `ConcurrentDictionary` that is currently in use.
This block of code parses out the Resource Name and Operation Name from the `TraceAttribute` and places the same defaults the `TraceAnnotationIntegration` uses.

Reply:
Much appreciated, thank you!! 🙇🏼